pymultimatic/model/zone.py

```diff
@@ -26,14 +26,19 @@ class ZoneHeating(Function):
         OperatingModes.OFF,
         OperatingModes.DAY,
         OperatingModes.NIGHT,
+        OperatingModes.NORMAL,
         OperatingModes.QUICK_VETO,
+        OperatingModes.REDUCED,
+        OperatingModes.TIME_CONTROLLED,
     ]
 
     def _active_mode(self) -> ActiveMode:
         if self.operating_mode == OperatingModes.OFF:
             mode = ActiveMode(constants.FROST_PROTECTION_TEMP, OperatingModes.OFF)
         elif self.operating_mode == OperatingModes.DAY:
             mode = ActiveMode(self.target_high, OperatingModes.DAY)
+        elif self.operating_mode == OperatingModes.MANUAL:
```

Comment:
Should it also checks if this is REDUCED mode ?

Reply:
Yes, this can be done. I did not need to do this because the REDUCTED mode is the time slot not defined during TIME_CONTROLLED mode. I assumed it was NIGHT mode for compatibility reasons. (see model/mapper.py L.247)
But there is nothing to stop it being not supported.

Reply:
I tried from the Senso application and from the MiPro Sense module but I could not find the case where the REDUCTED operation mode is transmitted. For SENSO, there are only the modes: OFF, MANUAL and TIME_CONTROLLED.
The term REDUCTED is used in the application to configure the unprogrammed temperature of the zones but is never indicated in the API responses.